docs/developers/components/finder.rst

```diff
@@ -0,0 +1,328 @@
+#doctest: +ELLIPSIS
+.. _translation_file_finder:
+
+
+Translation File Finder
+=======================
+
+This tool provides a way of mapping translation files in a filesystem to pootle codes for
```

Comment:
i did mean codes, but not sure what is clearest - ie it doesnt return a `pootle_path` more like the contents of ``split_pootle_path``